doc/development.md

```diff
@@ -221,10 +222,14 @@ helm unittest -u deploy/helm
 To enable code completion when writing new tests, install a VS Code extension providing a YAML Language server, like `redhat.vscode-yaml`.
 
 ## Integration tests
-Integration tests are located in `tests/integration` and are supposed to verify if metric processing is delivering expected outcome.
+Integration tests are located in `tests/integration` and are supposed to verify if metric processing is delivering expected outcome. 
+
+### Timeseries Mock Storage
+Important part of integration test is storage of telemetry data in ClickHouse. It is deployed as part of `skaffold dev`. In order to manually look at the data stored in the Clickhouse use Copilot Prompt `/query-clickhouse` (e.g. `/query-clickhouse show me what relationship state event types were ingested`). 
```

Comment:
I'd also mention that for running queries manually, one can use the ClickHouse web UI available at `http://localhost:8123/play`.

Reply:
Hm I actually had no idea that this web UI Clickhouse has. I mentioned it there